src/components/Header/index.tsx

```diff
@@ -1,8 +1,3 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
```

Comment:
Do we need to remove it? I believe Pierce added this (not completely sure though who did). This is in every file.

Reply:
I think we can leave it for now!